configure

```diff
@@ -172,6 +172,12 @@ parser.add_option('--openssl-use-def-ca-store',
     dest='use_openssl_ca_store',
     help='Use OpenSSL supplied CA store instead of compiled-in Mozilla CA copy.')
 
+parser.add_option('--openssl-system-ca-path',
+    action="store",
```

Comment:
Could you use `action='store'` (single quotes) here?

Reply:
No problems, I'll update that. Thanks